altsrc/helpers_test.go

```diff
@@ -1,13 +1,23 @@
 package altsrc
 
 import (
+	"os"
 	"reflect"
+	"runtime"
+	"strings"
 	"testing"
 )
 
+var (
+	wd, _ = os.Getwd()
```

Comment:
I would put that into the `expect` function below and check for the error. (non blocking)